source/src/ZombieWalker.kt

```diff
@@ -0,0 +1,7 @@
+class ZombieWalker(private val velocity: Int = 1) : IWalker {
+    override fun walk() {
+        val speedOfZombieWalker: Speed = Speed(velocity)
+        println("I`m a zombie walker, and my ")
+        speedOfZombieWalker.showSpeedConsole()
```

Comment:
So, now i can see that it is not comfortable design of this program.
You need make class instance, feed the variables there, and call method's of this object.
Why not just use a static method that gets the speed and prints it out?